src/phoenix/experimental/evals/models/openai.py

```diff
@@ -287,3 +292,15 @@ def get_tokens_from_text(self, text: str) -> List[int]:
 
     def get_text_from_tokens(self, tokens: List[int]) -> str:
         return self.encoder.decode(tokens)
+
+    @property
+    def supports_function_calling(self) -> bool:
+        if (
+            self._is_azure
+            and self.openai_api_version
+            and self.openai_api_version[:10] < "2023-07-01"
+        ):
```

Comment:
docs: add where you got this info from - it would be helpful for future readers to understand where this comes from  if they ever need to make adjustments.

Reply:
ok will add a comment here